features/gba-dev/verblijfstitel.feature

```diff
@@ -0,0 +1,91 @@
+#language: nl
+
+@post-assert
+Functionaliteit: Verblijfstitel
+
+Achtergrond:
+  Gegeven landelijke tabel "Verblijfstitel" heeft de volgende waarden
+  | code | omschrijving                                                                   |
+  |   00 | Onbekend                                                                       |
+  |   09 | Art. 9 van de Vreemdelingenwet                                                 |
+  |   37 | Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
+
+  Scenario: verblijfstitel heeft geen verblijfstitel
```

Comment:
Is de titel niet een beetje cryptisch? 
Verblijfstitel heeft geen gegevens over de verblijfstitel? Of: verblijfstitel bevat geen gegevens? 

Heb het even opgezocht: iedereen die in Nederland woont of zich in Nederland vestigt, heeft een verblijfstitel. Hierin staat bijvoorbeeld dat u de Nederlandse nationaliteit heeft of wat uw verblijfsstatus is. Dit kunt u aantonen met een geldig identiteits- of verblijfsdocument. 

Eigenlijk zouden we nationaliteit hier in moeten verwerken. Juiste interpretatie van verblijfstitel vereist domeinkennis. Niet voor nu, maar misschien iets voor de geplande indicatie "rechtmatig in NL"

Reply:
"verblijfstitel heeft geen verblijfstitel" zou moeten worden "persoon heeft geen verblijfstitel"

Eens met Cathy dat rechtmatig verblijf en/of een samenvoeging van nationaliteit en verblijfstitel (verblijfsvergunning) op basis waarvan met bepaalde rechten heeft. Dat zou je ook kunnen aanvullen met andere daarvoor relevante gegevens, zoals duur verblijf in Nederland (op basis van immigratiedatum). Dan hoef je wellicht ook niet de nationaliteit te weten, maar of je EU of EER inwoner bent. En niet welke verblijfstitel, maar alleen het type (I, II, III, of IV)

Reply:
Titel aangepast naar Persoon heeft geen verblijfstitel. Opmerkingen over het meenemen van de nationaliteit in het bepalen van de verblijfstitel opgenomen in een nieuwe issue die op de backlog staat.